src/utils.ts

```diff
@@ -33,5 +48,12 @@ export const throwFn = (e: Error): never => {
  * @returns A middleware function that handles errors thrown by the endpoint function.
  */
 export const asyncCatch =
-  (fn: EndpointFn) => (req: Request, res: Response, next: NextFunction) =>
+  (fn: (req: Request, res: Response) => Promise<void>) =>
+  (req: Request, res: Response, next: NextFunction) =>
     fn(req, res).catch(next);
+
+export const expressHandler =
+  (handler: Handler) => (req: Request, res: Response, next: NextFunction) =>
+    handler(Object.assign({}, req.query, req.params))
```

Comment:
Since we don't have post requests, I've passed all of the query and url parameters to the express endpoints.